src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/LookupTable/LookupTable.hooks.ts

```diff
@@ -29,7 +40,29 @@ export const useSubscaleLookupTableSetup = ({
       return FileError.SchemaValidation;
     }
 
-    const mappedData = file.data.map(processImportedData);
+    const mappedData = file.data.map((item: Record<string, string | number>) => {
+      Object.keys(item).forEach(
+        (k) => (item[k] = typeof item[k] === 'string' ? (item[k] as string).trim() : item[k]),
+      );
+
+      const mappedItem: LookupTableDataItem = {
+        ...(item as LookupTableDataItem),
+        sex: (item.sex as string) || null,
+        id: uuidv4(),
+      };
+
+      if (featureFlags.enableCahmiSubscaleScoring) {
+        mappedItem.severity = (item.severity as TScoreSeverity) || null;
+      }
+
+      return mappedItem;
+    });
+
+    const incompleteSeverityData = mappedData.some((item) => !item.severity);
+    if (incompleteSeverityData) {
+      setWarning(warnings.incompleteSeverityData);
+    }
```

Comment:
Should these lines also be behind the feature flag? I'm getting this when it's turned off, and I upload a CSV that lacks the `severity` column:
![CleanShot 2024-09-16 at 17 32 15@2x](https://github.com/user-attachments/assets/12709ab1-51af-46bb-8e7c-fb778f3d5816)

Reply:
Thanks for catching this Paul 🙏